airflow/providers/google/cloud/hooks/bigquery.py

```diff
@@ -3086,30 +3086,19 @@ async def get_job_instance(
         with await self.service_file_as_context() as f:
             return Job(job_id=job_id, project=project_id, service_file=f, session=cast(Session, session))
 
-    async def get_job_status(
-        self,
-        job_id: str | None,
-        project_id: str | None = None,
-    ) -> str | None:
-        """
-        Polls for job status asynchronously using gcloud-aio.
-
-        Note that an OSError is raised when Job results are still pending.
-        Exception means that Job finished with errors
-        """
+    async def get_job_status(self, job_id: str | None, project_id: str | None = None):
```

Comment:
Why'd you get rid of the return type annotation?

Reply:
aha, sorry.  actually we should keep it and I'll restore

Reply:
see -> 06f27e3399e96b19ef197f6e3bd0f816eeaa538e